File: src/net/eduard/permissions/command/PermissionsReloadCommand.java

package net.eduard.permissions.command;

import org.bukkit.command.Command;
import org.bukkit.command.CommandSender;

import net.eduard.api.lib.manager.CommandManager;
import net.eduard.permissions.EduPermissions;

public class PermissionsReloadCommand extends CommandManager {

	public PermissionsReloadCommand() {
		super("reload","recarregar");
		setDescription("Recarregar as permissões e grupos");
	}
	@Override
	public boolean onCommand(CommandSender sender, Command command, String label, String[] args) {
		EduPermissions.getInstance().reload();
		sender.sendMessage("§aPlugin EduPermissions salvado com sucesso");
	return true;
	}
	

}

File: src/main/java/net/eduard/permissions/command/PermissionsReloadCommand.kt
package net.eduard.permissions.command

import net.eduard.api.lib.command.Command
import net.eduard.api.lib.command.Sender
import net.eduard.permissions.api.PermissionsAPI

class PermissionsReloadCommand : Command("reload", "recarregar") {
    override fun onCommand(sender: Sender, args: List<String>) {
        PermissionsAPI.plugin.reload()
        sender.sendMessage("§aPlugin EduPermissions salvado com sucesso")
    }

    init {
        description = "Recarregar as permissões e grupos"
    }
}